auditbeat/docs/fields.asciidoc

```diff
@@ -3042,6 +3035,184 @@ type: object
 Image labels.
 
 
+--
+
+[[exported-fields-ecs]]
+== ECS fields
+
+Contains ECS fields (https://github.com/elastic/ecs)
+
+
+
+*`container.runtime`*::
```

Comment:
Are we using the `container` fields in any beat already? If yes, I would expect them to be removed somewhere. If not, should we already add them?